src/main/resources/servers/terra-verily-dev.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "terra-verily-dev",
-  "description": "Terra development environment deployed at Verily.",
+  "description": "Verily dev environment",
 
   "dataRepoUri": "https://jade.datarepo-dev.broadinstitute.org",
-  "samUri": "https://terra-autopush-sam.api.verily.com",
-  "workspaceManagerUri": "http://terra-autopush-wsm.api.verily.com"
+  "samUri": "https://sam-dev.api.verily.com",
+  "workspaceManagerUri": "https://wsm-dev.api.verily.com",
+  "wsmDefaultSpendProfile": "zloery-spend-profile"
```

Comment:
@zloery Could you confirm these URLs and spend profile name? I think the WSM one, in particular, may be wrong.

Reply:
The Sam one is right for Verily dev, but WSM is at `https://terra-dev-workspace.api.verily.com`. no idea what's going on with URL consistency

Reply:
Updated, thanks.